src/twisted/python/_typing.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+"""
+Short-hands for L{typing}.
+"""
+
+from typing import Union
+
+String = Union[bytes, str]
```

Comment:
We could use [`typing.AnyStr`](https://docs.python.org/3/library/typing.html?highlight=typing#typing.AnyStr) instead.

Reply:
Yeah, let's not do this.

Reply:
(I'd much rather just inline `Union[bytes, str]` if we're using it in a context where a typevar doesn't work.  Having this as an alias makes it too convenient to reach for.)

Reply:
I deleted this file.